manifests/4.6/logging.openshift.io_clusterloggings_crd.yaml

```diff
@@ -30,11 +30,6 @@ spec:
             submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
           type: string
         metadata:
-          properties:
-            name:
-              enum:
-              - instance
-              type: string
```

Comment:
This should not be dropped when generating CRDs.

Reply:
Weird, calling `make generate` actually drop these lines